Cocoapods Support?

I'm interested in using this library for a prototype Swift iOS application. It's nice that https://github.com/apple/swift-protobuf allows for installation. I was wondering if it would be possible to offer the SwiftGRPC.xcodeproj through Cocoapods, though I suppose it would require BoringSSL, CgRPC, gRPC, and CzLib all be available through Cocoapods as well.

@MrMage I tried adding your branches as dependencies in my pod file but was unable to install them because I don't believe they're published to the master specs repo. I then attempted to clone your forks and push your pod specs to my private repo, and the gRPC-Core spec didn't validate. This was the output of the failed validation:
```
Validating spec
 -> gRPC-Core (1.9.1)
    - ERROR | [gRPC-Core/Implementation, gRPC-Core/Cronet-Implementation, gRPC-Core/Tests, and more...] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - ERROR | xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release-iphonesimulator/gRPC-Core/grpc.framework/Headers/support/log.h:22:10: error: include of non-modular header inside framework module 'grpc': '/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/9.0.0/include/inttypes.h' [-Werror,-Wnon-modular-include-in-framework-module]
    - WARN  | xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release-iphonesimulator/gRPC-Core/grpc.framework/Headers/support/subprocess.h:31:51: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release-iphonesimulator/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:40:18: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release-iphonesimulator/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:42:26: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release-iphonesimulator/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:44:25: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release-iphonesimulator/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:46:38: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - NOTE  | [gRPC-Core/Implementation,gRPC-Core/Tests] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/App/main.m:3:9: fatal error: could not build module 'grpc'
    - ERROR | [OSX] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release/gRPC-Core/grpc.framework/Headers/support/log.h:22:10: error: include of non-modular header inside framework module 'grpc': '/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/9.0.0/include/inttypes.h' [-Werror,-Wnon-modular-include-in-framework-module]
    - WARN  | [OSX] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release/gRPC-Core/grpc.framework/Headers/support/subprocess.h:31:51: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | [OSX] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:40:18: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | [OSX] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:42:26: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | [OSX] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:44:25: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - WARN  | [OSX] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release/gRPC-Core/grpc.framework/Headers/impl/codegen/fork.h:46:38: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    - ERROR | [gRPC-Core/Implementation,gRPC-Core/Tests] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release-iphonesimulator/gRPC-Core/grpc.framework/Modules/module.modulemap:3:19: error: umbrella header 'grpc.h' not found
    - ERROR | [gRPC-Core/Implementation,gRPC-Core/Tests] xcodebuild:  /var/folders/9j/zzwbsh056dx9ddrwb3b_jbrh0000gn/T/CocoaPods-Lint-20180220-13301-1jmikt0-gRPC-Core/DerivedData/App/Build/Products/Release/gRPC-Core/grpc.framework/Modules/module.modulemap:3:19: error: umbrella header 'grpc.h' not found
    - NOTE  | [gRPC-Core/Cronet-Implementation] xcodebuild:  clang: error: linker command failed with exit code 1 (use -v to see invocation)
    - WARN  | [gRPC-Core/Tests] xcodebuild:  gRPC-Core/test/core/end2end/tests/streaming_error_response.cc:239:14: warning: variable 'status' may be uninitialized when used here [-Wconditional-uninitialized]
    - NOTE  | [gRPC-Core/Tests] xcodebuild:  gRPC-Core/test/core/end2end/tests/streaming_error_response.cc:114:3: note: variable 'status' is declared he
```

--- a/SwiftGRPC.podspec
+++ b/SwiftGRPC.podspec
@@ -45,7 +45,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Sources/SwiftGRPC/*.swift', 'Sources/SwiftGRPC/**/*.swift', 'Sources/CgRPC/shim/*.[ch]'
   s.public_header_files = 'Sources/CgRPC/shim/cgrpc.h'
 
-  s.dependency 'gRPC-Core', '~> 1.9.1'
-  s.dependency 'BoringSSL', '~> 9.1'
-  s.dependency 'SwiftProtobuf', '~> 1.0.2'
+  s.dependency 'gRPC-Core', '~> 1.11.0-pre2'
+  s.dependency 'BoringSSL', '~> 10.0'
+  s.dependency 'SwiftProtobuf', '~> 1.0.3'
 end

